common/util/format.ts

```diff
@@ -40,12 +40,22 @@ export function manaToUSD(mana: number) {
   })
 }
 
+function getPercentDecimalPlaces(zeroToOne: number) {
+  return zeroToOne < 0.02 || zeroToOne > 0.98 ? 1 : 0
+}
+
 export function formatPercent(zeroToOne: number) {
   // Show 1 decimal place if <2% or >98%, giving more resolution on the tails
-  const decimalPlaces = zeroToOne < 0.02 || zeroToOne > 0.98 ? 1 : 0
+  const decimalPlaces = getPercentDecimalPlaces(zeroToOne)
   return (zeroToOne * 100).toFixed(decimalPlaces) + '%'
 }
 
+export function formatPercentNumber(zeroToOne: number) {
+  // Show 1 decimal place if <2% or >98%, giving more resolution on the tails
+  const decimalPlaces = getPercentDecimalPlaces(zeroToOne)
+  return Math.round((zeroToOne * 100 * 1) ^ decimalPlaces) / (1 ^ decimalPlaces)
```

Comment:
This is a divide by zero error! Remember, ^ is XOR not exponentiation (which is ** in typescript)

Reply:
This function doesn't make that much sense even with exponents. Don't you just want
`(zeroToOne * 100).toFixed(decimalPlaces)`  without the percent sign at the end?

Reply:
Oh it has to be a number to have the cool counting down/up effect when the probability changes

Reply:
I just converted it and used toFixed